Automattic/jetpack
# CLI: Add jetpack build packages command

- Adds `jetpack build packages` as a way to build all packages.

#### Changes proposed in this Pull Request:

* Adds `jetpack build packages`.

#### Jetpack product discussion



p1615222710133700-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?


n/a

#### Testing instructions:






* `jetpack build plugins/jetpack --production` ensure that projects/plugins/jetpack/vendor only have prod depds and that lazy-images and jitm packages are built.
* `jetpack build packages` -- packages all build?
